src/features/user/Account/AccountHistory.module.scss

```diff
@@ -271,6 +270,41 @@
   color: $primaryFontColor;
 }
 
+.issueButton {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  background-color: $primaryColor;
+  transition-duration: 0.4;
+  transition-timing-function: ease-in;
+  padding: 8px 12px;
+  border-radius: 100px;
+  margin: 25px 0px 10px;
+  white-space: nowrap;
+  font-family: $primaryFontFamily;
+  cursor: pointer;
+  color: #fff;
+}
+
+.issueButtonMobile {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  background-color: $primaryColor;
+  transition-duration: 0.4;
+  transition-timing-function: ease-in;
+  padding: 8px 12px;
+  border-radius: 100px;
+  margin: 0px 0px 10px;
+  white-space: nowrap;
+  font-family: $primaryFontFamily;
+  cursor: pointer;
+  color: #fff;
+  width: 100%;
+}
+
```

Comment:
Should we use Mui button styles here (with customizations if necessary)?

We're currently using a mishmash of button styles in the app.

Reply:
Fixed here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1677/commits/bff0d19a7d796e72ba4a2d6bb4d5cd7896e8778d

Reply:
Fixed here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1677/commits/bff0d19a7d796e72ba4a2d6bb4d5cd7896e8778d